image_cloudevents/modelmonitor.ipynb

```diff
@@ -0,0 +1,303 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%store -r\n",
+    "import time\n",
+    "import os,json\n",
+    "import requests\n",
+    "from dkube.sdk import *\n",
+    "from dkube.sdk.api import DkubeApi\n",
+    "from dkube.sdk.rsrcs import DkubeModelmonitor\n",
+    "from dkube.sdk.rsrcs.modelmonitor import DatasetClass,ModelType,DriftAlgo\n",
+    "from dkube.sdk.rsrcs.modelmonitor import DatasetFormat,DkubeModelmonitoralert, TimeZone\n",
+    "from dkube.sdk.rsrcs.modelmonitor import DataType, ChannelOrder, ImageDataSavedFileFormat"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def wait_for_deployment_running(deployment_id):\n",
+    "    status = None\n",
+    "    inference_url, inference = None, None\n",
+    "    while True:\n",
+    "        data = api.get_deployment(deployment_id)\n",
+    "        status = data.data.inferenceservice_deployment.parameters.generated.status.state\n",
+    "        inference = data.data.inferenceservice_deployment.parameters.inference\n",
+    "        inference_url = data.data.inferenceservice_deployment.parameters.generated.details.serving.servingurl\n",
+    "        if status == \"RUNNING\":\n",
+    "            break\n",
+    "        print(\"waiting for deployment to be running\")\n",
+    "        time.sleep(api.wait_interval)\n",
+    "    return inference, inference_url"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "api = DkubeApi(URL=os.getenv('DKUBE_URL'),token=os.getenv(\"DKUBE_USER_ACCESS_TOKEN\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Enabling logs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "while True:\n",
+    "    DEPLOYMENT_ID = api.get_deployment_id(name=MONITOR_NAME)\n",
+    "    if DEPLOYMENT_ID:\n",
+    "        break\n",
+    "    print(\"waiting for deployment to come up\")\n",
+    "    time.sleep(api.wait_interval)\n",
+    "inference, INFERENCE_URL = wait_for_deployment_running(DEPLOYMENT_ID)\n",
+    "if not inference.enable_logs:\n",
+    "    print(\"Enabling logs\")\n",
+    "    serving = DkubeServing(user=DKUBEUSERNAME, name=MONITOR_NAME)\n",
+    "    serving.update_enable_logs(enable_logs=True)\n",
```

Comment:
We are auto enabling logs if it is local deployment. If this example is for local deployments & cloudevents, we can ignore this

Reply:
okay, it will be removed.